File: Estructura_de_datos_Python/seleccion.py
array = [64, 25, 12, 22, 11]
print("Antes de ordenar:", array)

for i in range(len(array)):
    minimo = i
    for j in range (i+1, len(array)):
        if array[minimo] > array[j]:
            minimo = j
    aux = array[i]
    array[i] = array[minimo]
    array[minimo] = aux

print("Después de ordenar:", array)

File: Ordenamiento_JS/seleccion.js
let array = [5, 3, 8, 4, 2];

console.log("Antes de ordenar:", array);

for (let i = 0; i < array.length; i++) {
    let minimo = i;
        for (let j = i + 1; j< array.length; j++) {
            if (array[minimo] > array[j]){
                minimo = j;
            }
        }
        let aux = array [i];
        array[i] = array[minimo];
        array[minimo] = aux;
 }
console.log("Después de ordenar:", array);